test/functional/sqlsrv/MsHelper.inc

```diff
@@ -325,6 +377,7 @@ function connect($options = array(), $disableCE = false)
  * @param  object  $conn : sqlsrv connection object
  * @param  string  $tbname : name of the table to be created
  * @param  array  $columnMetaArr : array of ColumnMeta objects, which contain metadata for one column
+ * @return object sqlsrv statement
```

Comment:
I believe sqlsrv statement is a resource not an object, same with sqlsrv connection